components/RadioGroup/styles.ts

```diff
@@ -0,0 +1,41 @@
+import styled, { css } from 'styled-components';
+import COLORS from '@/styles/colors';
+
+interface SelectedProps {
+  $isSelected: boolean;
+}
+
+export const ComponentContainer = styled.button<SelectedProps>`
+  display: flex;
+  flex-direction: row;
+  gap: 0.9375rem;
```

Comment:
i just realized- we should be using px for non-text values. rem should only be used for text so that it responds dynamically to the user's default size. but we don't want the gaps to also increase with the user's default text size